oat-sa/extension-tao-lti
# feat: LTI 1.3 Platforms & registration repo implemented

## Ticket
https://oat-sa.atlassian.net/browse/TR-879

## Summary
LTI 1.3 Platforms + Registration repo implemented.

## How to test
At the moment, it's not possible to test the whole workflow including the registration repository, as LTI1.3 isn't implemented, yet.
The only way is to check the UI screen is correct. Here's how to do that:
1. Log in to the BO.
2. Click the 'Gear' icon to open preferences, select 'LTI 1.3 Platform' which should show up with this PR applied.
3. Play around with CRUD on platforms etc, check the validation is implemented correctly.

## Sandbox environment
Deployed on http://cgen-ltiregistrations.playground.kitchen.it.taocloud.org:42141/
